certified-connectors/GISCloud/apiProperties.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#0b86cc",
+    "scriptOperations": [
+        "getLayer"
```

Comment:
@arkore please update the operation name if you'll be changing the operationId in apiDefinition file.

Reply:
I have updated the operationId to conform with the Pascal Case standard in alignment with the apiDefinition file